sys/include/net/gnrc.h

```diff
@@ -155,26 +155,46 @@
  * Transmitting Packets
  * --------------------
  *
- * To relay the @ref net_gnrc_pkt to interested threads the @ref
- * net_gnrc_netapi offers a dispatch functionality. The following example
+ * A packet is transmitted by relaying it to threads interested in handling (and
+ * dispatching) packets of its type. To do this, the @ref
+ * net_gnrc_netapi offers a dispatch functionality called @ref
+ * gnrc_netapi_dispatch_send().
```

Comment:
`.. and gnrc_netapi_dispatch_receive()`. Also I would prefer that it is pointed out that these are only helper functions. One can as well use raw `msg_send()` if they know what they are doing.